test/test_accelerator.py

```diff
@@ -156,6 +157,84 @@ def test_generic_event_behavior(self):
         ):
             event1.elapsed_time(event2)
 
+    @unittest.skipIf(TEST_MPS, "MPS doesn't support torch.accelerator memory API!")
```

Comment:
Can you open an issue for it?
I guess it's because we need to refactor the caching allocator for them to use it there as well?

Reply:
OK, I open an issue here https://github.com/pytorch/pytorch/issues/156805